src/docs/teams/deployments/sandboxes.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Teams | Sandboxes
+layout: docs.hbs
+---
+# Sandboxes
+
+<p class="alert alert-warning"><i class="fad fa-flask"></i> Thank you so much for participating in the <strong>Truffle Teams Private Preview</strong>! Your access gets exclusive access to this feature in progress! Since we're still getting the recipe perfect, you may run into issues. <strong>Please let us know if you encounter any issues!</strong></p>
```

Comment:
the `Please let us know if you encounter any issues!` should be a hyperlink to https://truffle-blockstars.slack.com/archives/GUUKVMTLK